src/components/Form/index.tsx

```diff
@@ -188,245 +188,275 @@ export const Form = ({
-                    <Controller
-                      name='stars'
-                      control={control}
-                      rules={{ required: { value: true, message: 'rating is required' } }}
-                      render={({ field: { onChange, value }, fieldState: { error } }) => (
-                        <Tooltip title='A rating associated with the claim' placement='right' arrow>
-                          <FormControl sx={{ ml: 1, mr: 1, width: '22ch' }} fullWidth error={!!error}>
-                            <Typography>Review Rating</Typography>
-                            <Rating
-                              name='stars'
-                              value={value}
-                              onChange={(e, newValue) => onChange(newValue)}
-                              precision={1}
-                              size='large'
-                            />
-
-                            <FormHelperText>{error?.message}</FormHelperText>
-                          </FormControl>
-                        </Tooltip>
-                      )}
-                    />
-                  </>
-                ) : (
-                  <Tooltip title='If you want to add any additional site belongs to you' placement='right' arrow>
-                    <TextField
-                      {...register('object')}
-                      sx={{ ml: 1, mr: 1, width: '22ch' }}
-                      margin='dense'
-                      variant='outlined'
-                      fullWidth
-                      label='Object'
-                      key='object'
-                      type='text'
-                    />
-                  </Tooltip>
-                )}
-              </>
-            )}
-            <LocalizationProvider dateAdapter={AdapterDateFns}>
-              <DatePicker
-                label='Effective Date'
-                value={watchEffectiveDate}
-                onChange={(newValue: any) => setValue('effectiveDate', newValue)}
-                renderInput={(params: any) => (
-                  <TextField {...params} sx={{ ml: 1, mr: 1, width: '100%' }} variant='filled' />
-                )}
-              />
-            </LocalizationProvider>
-          </Box>
-        </form>
-      </DialogContent>
-      <DialogActions sx={{ display: 'flex', justifyContent: 'flex-end', width: '100%', columnGap: 3 }}>
-        <Button
-          onClick={onSubmit}
-          variant='contained'
-          size='large'
-          sx={{
-            ml: 1,
-            mr: 1,
-            width: '50%',
-            bgcolor: 'praimary.main',
-            margin: '0 auto',
-            '&:hover': {
-              backgroundColor: '#00695f'
-            }
-          }}
-        >
-          Submit
-        </Button>
-        {!!onCancel && <Button onClick={onCancel}>Cancel</Button>}
-      </DialogActions>
+                  )}
+                </>
+              )}
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
there are an issue in the date , it is not working , please find the problem , it should be here